src/power_grid_model_io/config/excel/vision_en.yaml

```diff
@@ -424,6 +424,11 @@ grid:
       extra:
         - ID
         - Name
+      exclude_filter:
+        - power_grid_model_io.functions.exclude_filters.exclude_all_columns_empty_or_zero:
```

Comment:
It shall `exclude all objects where P and Q is empty or zero value`
What it evaluates to is irrelevant to the user. (For creating the mask, these exclude_filters should evaluate to False for a match)

Everything is included by default. So include filters is not needed.

Reply:
> It shall `exclude all objects where P and Q is empty or zero value`

I broke it down the other way around: It shall `"exclude": ["all objects where P and Q is empty or zero value"]`

> What it evaluates to is irrelevant to the user. (For creating the mask, these exclude_filters should evaluate to False for a match)

If so, I would just go with `filters` instead of `exclude_filter` and let it be an array of filters. Then, the filter `exclude_all_columns_empty_or_zero` would indeed do just that.

That's also more portable than adding just 1 filter option, because a user may want to add multiple `exclude`'s and maybe even `include_column_nonzero`. Otherwise, adding this feature would be a breaking change later on.

> Everything is included by default. So include filters is not needed.

That's only for now. I'm also thinking about the transformer tap regulator:

```yaml
"include_filters":
  - "checkbox_checked": "Transformer.regulated"
  - "checkbox_checked": "ThreeWindingTransformer.regulated"
```

or

```yaml
"filters":
  - "include_checkbox_checked": "Transformer.regulated"
  - "include_checkbox_checked": "ThreeWindingTransformer.regulated"
```

which reads more obviously than

```yaml
"exclude_filters":
  - "exclude_checkbox_unchecked": "Transformer.regulated"
  - "exclude_checkbox_unchecked": "ThreeWindingTransformer.regulated"
```

Reply:
We don't have to complicate it. A single argument modification is applies to modification of 40 arguments in the code. Excel converter is temporary anyway. Its good to be least invasive.